geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionSingleHopDUnitTest.java

```diff
@@ -1501,11 +1482,11 @@ public String getId() {
     }
 
     @Override
-    public void execute(FunctionContext context) {
+    public void execute(FunctionContext<T> context) {
       RegionFunctionContext rc = (RegionFunctionContext) context;
       Region<Object, Object> r = rc.getDataSet();
-      Set filter = rc.getFilter();
-      if (rc.getFilter() == null) {
+      Set<Object> filter = (Set<Object>) rc.getFilter();
+      if (filter == null) {
         for (int i = 0; i < 200; i++) {
           r.put(i, i);
         }
```

Comment:
The `equals()` method for the Order class below is broken. Line 1604 should be:
```
      if (!(obj instanceof Order)) {
```

Reply:
Ok, I switched Object to ?